lsp-mode.el

```diff
@@ -1917,14 +1923,19 @@ WORKSPACE is the workspace that contains the progress token."
                                                 (replace-regexp-in-string "[\n\t ]+" " "))
                                            'face lsp-modeline-code-actions-face))
           (single-action? (= (length actions) 1))
+          (execute-code-actions-binding (->> 'lsp-execute-code-action
+                                             (where-is-internal)
+                                             (-find (lambda (o)
```

Comment:
can you test that with lsp-execute-code-action not bound to keybinding?

Reply:
It will return empty string

Reply:
then we dont need the () in the format call bellow.

Reply:
Good idea, fixed it